addStudy.py

```diff
@@ -0,0 +1,101 @@
+# addStudy.py
+
+#!/usr/bin/python3
+
+# python addStudy.py -c rwa_configuration.yml -l
+# python addStudy.py -c rwa_configuration.yml -a "studyName"
+
+# addStudy.py
+#
+# Adding new studies to the database
+
+# Import our libraries
+import os
+import sys
+import argparse
+import yaml
+import psycopg2
+from include.calibrationLib import *
+
+# List all studies
+def get_studies(configuration, studName):
+    # return "List to populate"
+    sql = '''SELECT * FROM STUDY'''
+
+    return select(configuration["connection_string"], sql, {'name': None})
+
+# add studies
+def add_file(configuration, Name):
+    Name = str(Name)
+
+    sql = '''INSERT INTO study(Name) VALUES(%s) RETURNING id;'''
+
+    return select(configuration["connection_string"], sql, {'Name': Name})
+
+# connection
+def select(connectionString, sql, parameters):
+    # Open the connection
+    connection = psycopg2.connect(connectionString)
+
+    cursor = connection.cursor()
+    # add study
+    if (sql.find("INSERT") == 0):
+        cursor.execute(sql, (parameters['Name'],))
+        study_id = cursor.fetchone()[0]
+
+        connection.commit()
+
+        cursor.close()
```

Comment:
No. This is bad practice and is _very_ confusing to other developers over the long term. If a function is intended to select something from the database it should not contain a special branch for insert statements.